pageserver/src/http/routes.rs

```diff
@@ -1091,43 +1092,64 @@ pub fn make_router(
             let handler = $handler;
             #[cfg(not(feature = "testing"))]
             let handler = cfg_disabled;
-            handler
+
```

Comment:
do you need the extra new line here?

Reply:
Not much, just added it there to emphasize the |r| transformation.
Will remove that if any other changes are needed to be added.